### DISCUSSION TITLE
List where an entity is used

### DISCUSSION THREAD

**Discussion by droidgren** [frenck/spook#870] [Ideas]
If I rename an entity Spook will tell me where that entity was used because it missing. (In Automations, dashboards etc)
It would be nice if you could list this information whiteout the rename, if I don't want to disrupt the entity.
Thanks

**Comment by frenck** [comment]
That is unfortunately not functionality Spook can provide. It can interact with Home Assistant, but not change the full behavior.

It floats around searching for things and reports it in a side-channel.


../Frenck
